src/components/layout-components/localization-buttons.tsx

```diff
@@ -13,34 +14,29 @@ const LocalizationButtons = () => {
   /**
    * Method to handle locale change
    */
-  const handleLocaleChange = (
-    event: React.MouseEvent<HTMLElement>,
-    newLanguage: Language | null
-  ) => {
+  const handleLocaleChange = (_event: MouseEvent<HTMLElement>, newLanguage: Language | null) => {
     setLanguage(newLanguage as Language);
   };
 
   /**
    * Renders localization buttons
    */
   const renderLocalizationButtons = () => (
-    <div>
-      <ToggleButtonGroup
-        value={language}
-        exclusive
-        onChange={handleLocaleChange}
-        aria-label="localization"
-      >
-        {strings.getAvailableLanguages().map((availableLanguage, idx) => {
-          const value = strings.getString(`localization.${availableLanguage}`, language);
-          return (
-            <ToggleButton value={availableLanguage} key={`${idx}`}>
-              {value}
-            </ToggleButton>
-          );
-        })}
-      </ToggleButtonGroup>
-    </div>
+    <ToggleButtonGroup
+      value={language}
+      exclusive
+      onChange={handleLocaleChange}
+      aria-label="localization"
+    >
+      {strings.getAvailableLanguages().map((availableLanguage, idx) => {
+        const value = strings.getString(`localization.${availableLanguage}`, language);
+        return (
```

Comment:
Having a line break before a return value is a good general rule for code readability

Reply:
Added a line break.